Sources/FocusEntity/FocusEntity.swift

```diff
@@ -261,6 +265,7 @@ open class FocusEntity: Entity, HasAnchoring, HasFocusEntity {
     if self.currentAlignment != .none {
       // It is ready to move over to a new surface.
       recentFocusEntityPositions.append(position)
+      performAlignmentAnimation(to: raycastResult.worldTransform.orientation)
```

Comment:
Need to test this on non-LiDAR device, might make things all messed up, but on LiDAR with estimated planes allowed it looks pretty cool! Might make it a FocusEntityComponent option

Reply:
looks good to me on non-LiDAR, I'll leave it in